### ISSUE TITLE
Problem with the latest Nightly

### ISSUE THREAD

**Issue by leonardo-m** [rust-lang/rustup#2518]
If I try to compile with the latest Nightly (nightly-2020-10-08-x86_64-pc-windows-gnu), rustc acts a bit like rustup, and it tries to download another compiler somewhere. This is the strangest Nightly problem I've seen so far.

An example:

```rust
...>rustc -vV
info: syncing channel updates for 'nightly-2020-10-08-x86_64-pc-windows-gnu'
info: latest update on 2020-10-08, rust version 1.49.0-nightly (91a79fb29 2020-10-07)
info: downloading component 'cargo'
info: downloading component 'clippy'
info: downloading component 'rust-docs'
info: downloading component 'rust-mingw'
info: downloading component 'rust-std'
info: downloading component 'rustc'
info: downloading component 'rustfmt'
info: installing component 'cargo'
info: Defaulting to 500.0 MiB unpack ram
info: installing component 'clippy'
info: installing component 'rust-docs'
^C
```

**Comment by ehuss** [comment]
This is normal behavior if you have an [override](https://rust-lang.github.io/rustup/overrides.html), and the overridden toolchain isn't installed.  Check if you have a `rust-toolchain` file or a directory override.

**Comment by leonardo-m** [comment]
I'll take a look, thank you (but I have changed nothing on purpose in my configuration).

**Comment by rbtcollins** [comment]
If you don't interrupt it, but let it finish, does it compile ok? And does
it download again on the next run, or is it satisfied at that point?

**Comment by rbtcollins** [comment]
Oh also, do a 'rustup show'

**Comment by jon-chuang** [comment]
I experience weird behaviour. For `rustup show`:
```
...
active toolchain
----------------

nightly-2020-10-09-x86_64-unknown-linux-gnu (default)
rustc 1.49.0-nightly (3525087ad 2020-10-08)
```

but `cargo -V` outputs `cargo 1.48.0-nightly (9d1a4863a 2020-10-05)`

Further, I experience type inference failures on 1.48. I hope this is a temporary problem.

**Comment by leonardo-m** [comment]
The successive Nightly doesn't show this problem, so I'm going to close this issue down... I'll reopen it if the problem appears again. I have changed nothing in this PC, so I think just one Nightly had that problem.

**Comment by ehuss** [comment]
Cargo's version on nightly is not directly tied to rustc's version. It will get updated sometime next week.

**Comment by mj555** [comment]
Ubsubscribe



Von meinem Samsung Galaxy Smartphone gesendet.


-------- Ursprüngliche Nachricht --------
Von: leonardo-m <notifications@github.com>
Datum: 10.10.2020 19:04 (GMT+01:00)
An: rust-lang/rustup <rustup@noreply.github.com>
Cc: Subscribed <subscribed@noreply.github.com>
Betreff: Re: [rust-lang/rustup] Problem with the latest Nightly (#2518)


Closed #2518<https://github.com/rust-lang/rustup/issues/2518>.
